ui/src/actions/beamline.js

```diff
@@ -45,10 +45,6 @@ export function updateBeamlineHardwareObjectStateAction(data) {
   return { type: BL_UPDATE_HARDWARE_OBJECT_STATE, data };
 }
 
-export function setBeamlineAttribute(name, value) {
-  return updateBeamlineHardwareObjectAction({ name, value });
-}
-
```

Comment:
Obsolete — this was modifying only the local state. Updating a beamline attribute should always be done via `setAttribute` and corresponding API call.